src/main/java/com/vite/Application.java

```diff
@@ -1,19 +1,21 @@
 package com.vite;
 
+import com.vite.service.ViteClient;
 import org.apache.log4j.Logger;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-    
+
 @EnableWebMvc
-@ComponentScan({"com.vite.controllers"})
+@ComponentScan({"com.vite.service"})
 @SpringBootApplication
 public class Application {
     static final Logger LOGGER = Logger.getLogger(Application.class);
 
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
+        new ViteClient();
```

Comment:
What I mean: Why are you initializing a client here? If the test works, then the test works.